src/main/webapp/app/exercises/shared/exercise-detail-common-actions/non-programming-exercise-detail-common-actions.component.html

```diff
@@ -24,6 +24,16 @@
         <span class="d-none d-md-inline" jhiTranslate="artemisApp.courseOverview.exerciseDetails.instructorActions.submissions">Submissions</span>
     </a>
 
+    <!-- Example Submission -->
+    <a
+        *ngIf="exercise.isAtLeastInstructor && (exercise.type == 'modeling' || exercise.type == 'text')"
```

Comment:
Shouldn't this be allowed for editors too?

Reply:
yes, done.